src/platform/bizhawk/bizinterface.c

```diff
@@ -265,8 +286,8 @@ EXP bizctx* BizCreate(const void* bios, const void* data, int length, const over
 		return NULL;
 	}
 
-	memset(ctx->sram, 0xff, 131072);
-	ctx->sramvf = VFileFromMemory(ctx->sram, 131072);
+	memset(ctx->sram, 0xff, 131072 + 16);

```

Comment:
Is 0xff a reasonable default for rtc sram?

Reply:
It's not, hence the hack init call after the first resetinternal. Note the last 16 bytes are not a guarentee to be used for RTC, that's only for Pokemans / games with 128K flash (Bokai is a notable game that does not have 128K flash).

When BizGetSaveRam is called GBASavedataRTCWrite will be called, which ensures whereever mGBA is storing RTC will be up to date (and not the 0xff trash).

0 padding is not either, as the control register is saved and its proper initial state is 0x40 (0 or 0xff would tell the game rtc is dead).

Reply:
Why even set it in the first place then?

Reply:
mGBA doesn't appear to set it, and it doesn't handle a 0 filled save without resulting in game complaining rtc is dead (due to initial calloc call, not setting it would just give mGBA a 0 filled sramvf.

Reply:
I mean why extend the 0xFF memset to the last 16 bytes.

Reply:
A consistent initial buffer I guess? It's garbage either way.

Reply:
A consistent initial buffer I guess? It's garbage either way.

Reply:
Since the RTC isn't always in those 16 bytes, and there's not an easy way to tell at this point where it will be, I'm fine with leaving this alone.